commonknowledge/join
# feat: add Stripe to main join flow block

## Description
- Adds Stripe components to front-end
- Adds Stripe token handling to back-end JoinService

## Motivation and Context
Part of the work to add Stripe support to the plugin.

## How Can It Be Tested?
1. Enable Stripe and add the 2 stripe keys on the settings page
2. Modify the payment plans and save them to trigger Stripe product creation
3. Sign up using the Join Form Block
4. Verify in the logs that a subscription was created
